### ISSUE TITLE
Encoding issue with polis letter l on GS

### ISSUE THREAD

**Issue by adam3smith** [zotero/translators#1553]
https://scholar.google.com/scholar?cluster=249138857392321717&hl=en&oi=scholarr

reported: https://forums.zotero.org/discussion/70322/ive-notice-that-zotero-has-a-problem-with-polish-letter-l-k#latest

I'm suspecting this is a bibtex issue but haven't had time to look any closer

**Comment by zuphilip** [comment]
Yes, the BibTeX code `\l` would need to map to the Polish letter `ł`. Actually, this is part of #1073, which we might want to continue...
